docs/encyclopedia/xdr.mdx

```diff
@@ -14,6 +14,6 @@ XDR is specified in [RFC 4506](http://tools.ietf.org/html/rfc4506.html) and is s
 
 Since XDR is a binary format and not as widely known as simpler formats like JSON, the Stellar SDKs all include tools for parsing XDR and will do so automatically when retrieving data.
 
-In addition, the Horizon API server generally exposes the most important parts of the XDR data in JSON, so they are easier to parse if you are not using an SDK. The XDR data is still included (encoded as a base64 string) inside the JSON in case you need direct access to it. .X files
+In addition, the Horizon API server generally exposes the most important parts of the XDR data in JSON, so they are easier to parse if you are not using an SDK. The XDR data is still included (encoded as a base64 string) inside the JSON in case you need direct access to it.
```

Comment:
```suggestion
In addition, the Horizon API server generally exposes the most important parts of the XDR data in JSON, so they are easier to parse if you are not using an SDK. The XDR data is still included (encoded as a base64 string) inside the JSON in case you need direct access to it.

## .X files
```

I believe `.X files` is meant to be another heading on this page. The link that follows points to a directory several IDL files named something like `Stellar-something.x`.

Reply:
Ah! Will update